src/AppBundle/Resources/public/js/views/up-itemsfieldview.js

```diff
@@ -132,7 +132,7 @@ YUI.add('up-itemsfieldview', function (Y) {
 
             if (selector) {
                 jQuery(selector).EzTags({
-                    autocompleteUrl: '/tags/admin/field/autocomplete?',
+                    autocompleteUrl: '/admin/tags/admin/field/autocomplete?',
```

Comment:
Not a big fan of this solution...
What if the admin siteaccess is different?

Reply:
Right, I'm looking for better solution

Reply:
Please take a look at: https://symfony.com/doc/master/bundles/FOSJsRoutingBundle/usage.html